nss-day-cohort-41/nutshell-sea-rose
# Zachs temp branch

Summary:
Created a way for new users to register an account and save their info in the database.

Tasks:
Got the input field values for creating a new user
Created a factory function for making new user
Passed our parameters into the factory function inside the click event. And returned the targeted object.
Got that new user object to save inside of the json database. 

Type of change:
New feature (non-breaking change which adds functionality)

Testing Instructions:
Open up your browser (with dev tools open)
Type in the required info into the form fields in the Create Account section. 
Required fields are username, password, and email. 
Then click the "Create Account" button.
The console should log your data, and your account will be saved to the database.
